kobo/settings/base.py

```diff
@@ -1130,6 +1133,16 @@ def dj_stripe_request_callback_method():
     KOBOCAT_MEDIA_PATH = os.environ.get('KOBOCAT_MEDIA_PATH', '/srv/src/kobocat/media')
 
 
+STORAGES = {
+    'default': {
+        'BACKEND': default_file_storage,
+    },
+    'staticfiles': {
+        'BACKEND': 'django.contrib.staticfiles.storage.StaticFilesStorage',
```

Comment:
What happens if we omit staticfiles entirely? If that doesn't work, refer to from django.conf import global_settings